src/main/webapp/app/css/style.css

```diff
@@ -71,4 +76,14 @@ ul {
 
 #tags-field-1, #tags-field-2, #tags-field-3 {
     width: 200px;
+}
+
+#findGoalButton {
+    margin-right: 40%;
+    margin-bottom: 5%;
+}
+
+#findUserButton {
```

Comment:
These two button styles could be merged into one.

Reply:
merged into the one style